docs/usage.rst

```diff
@@ -67,20 +51,18 @@ Configuration
 
 The ``config`` dictionary must have at least the following items:
 
-==================== ======== ==================== ========================
-Key                  Type     Default              Other built-in options
-==================== ======== ==================== ========================
-namelist             Namelist default Namelist     none
-experiment_name      str      'default_experiment' n/a
-diag_table           str      'default'            'grid_spec', 'no_output'
-data_table           str      'default'            none
-initial_conditions   str      'gfs_example'        'restart_example'
-forcing              str      'default'            none
-==================== ======== ==================== ========================
-
-In addition to one of the built-in options, a custom ``diag_table`` and ``data_table`` can be specified
-by supplying a path to a file. Custom ``initial_conditions`` and ``forcing`` can be specified by
-supplying a path to a directory that contains the appropriate files. Paths to files or directories on the local
+==================== ======== ============================================
+Key                  Type     Description
+==================== ======== ============================================
+namelist             dict     Model namelist
+experiment_name      str      Name of experiment to use in output
+diag_table           str      location of diag_table file, or one of ("default", "grid_spec", "no_output")
+data_table           str      location of data_table file, or "default"
+initial_conditions   str      location of directory containing initial conditions data
+forcing              str      location of directory containing forcing data
+==================== ======== ============================================
```

Comment:
Need to add the `orographic_forcing` entry.

Reply:
Added.